uaa_bot/notifier.py

```diff
@@ -0,0 +1,112 @@
+from email.mime.text import MIMEText
+import os
+import smtplib
+import ssl
+
+from email_validator import validate_email, EmailNotValidError
+from jinja2 import Environment, PackageLoader, select_autoescape
+
+from uaa_bot import config, templates
+
+TEMPLATES = os.listdir(
+    os.path.join(os.path.dirname(os.path.realpath(__file__)), "./templates")
+)
+
+TEMPLATE_ENV = Environment(
+    loader=PackageLoader("uaa_bot", "templates"), autoescape=select_autoescape(["html"])
+)
+
+
+class Notifier:
+    def __init__(self, email: str, smtp_config=config.smtp):
+        self.smtp_config = smtp_config
+        self.email = email
```

Comment:
this should be self._email, no?

Reply:
oh, I see. Nevermind

Reply:
This tripped me up too - I saw the `@property` below but it took me a bit before I connected the `@email.setter` block.